infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java

```diff
@@ -144,7 +145,7 @@ public static SystemSchemaBuilderRule valueOf(final String databaseType, final S
     /**
      * Judge whether current table is system table or not.
      *
-     * @param schema schema
+     * @param schema    schema
```

Comment:
This appears to be an inadvertent modification

Reply:
It seems like so, should I make a new commit to rectify it?

Reply:
Just modify it directly on the branch you submitted

Reply:
done